requirements/base.txt

```diff
@@ -15,11 +15,9 @@ requests==2.18.4
 boto==2.48.0
 
 
-git+https://github.com/mysociety/django-pipeline-csscompressor
-
 # TODO: review this when you upgrade dc_base_theme to 0.3
-libsass==0.13.6  # pyup: >=0.13.6,<0.13.7
+libsass==0.13.7  # pyup: >=0.13.6,<0.14.0
```

Comment:
Does upgrading to libsass `>=0.14` do bad stuff with the 0.3 theme? If so, lets raise an issue on the theme repo. Whereas I wasn't particularly inclined to fix the legacy theme, I'm more inclined to care about this in the context of the theme we're actively developing/migrating projects to.

Reply:
Tested and it seems to work 👍